example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -1151,7 +1151,8 @@ describe('The S3 Ingest Granules workflow', () => {
           try {
             failOnSetupError([beforeAllError]);
 
-            file = granule.files.filter((x) => x.fileName.match(/\.hdf$/))[0];
+            file = granule.files.sort((a, b) => (a.key > b.key ? 1 : -1))[0];
```

Comment:
I'm not sure the file filter here isn't *better* in that it's more explicit what we're actually sorting for.   This looks like we solved this issue in the feature branch but not in main :facepalm:. 

I think we should consider retaining the fix from feature over this approach, even if both are valid.

Reply:
Yup, sure thing. https://github.com/nasa/cumulus/pull/3225/commits/1804e1b4ab7b1414df76df51fa9a656c0c98500c